src/SprykerEco/Yves/Braintree/Theme/default/components/molecules/braintree-credit-card/braintree-credit-card.ts

```diff
@@ -1,6 +1,6 @@
-import PaymentForm from '../payment-form/payment-form';
+import PaymentForm from '../braintree-payment-form/braintree-payment-form';
 
-export default class CreditCard extends PaymentForm {
+export default class BraintreeCreditCard extends PaymentForm {
```

Comment:
suggest to rename `PaymentForm` to `BraintreePaymentForm`

Reply:
fixed